src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -34,13 +36,20 @@
 public class ExtractCohortEngine {
     private final Logger logger;
 
+    private final List<LongRange> PARRegions = List.of(
+            new LongRange(23000000010001L, 23000002781479L),    // PAR1, X
+            new LongRange(24000000010001L, 24000002781479L),    // PAR1, Y
+            new LongRange(23000155701383L, 23000156030895L),    // PAR2, X
+            new LongRange(24000056887903L, 24000057217415L));   // PAR2, Y
```

Comment:
I don't question this is right but some comments and link(s) to authoritative sources would be appreciated.

Reply:
Linked to wikipedia page now

Reply:
This is for hg38. If you don't intend to make it support other builds (which is fine), could you create a ticket to add support for other builds?